release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2872,6 +2872,42 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.13
+[id="ocp-4-17-13_{context}"]
+=== RHSA-2025:0351 - {product-title} {product-version}.13 bug fix, and security update advisory
+
+Issued: 21 January 2025
+
+{product-title} release {product-version}.13 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:0351[RHSA-2025:0351] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:0354[RHBA-2025:0354] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.13 --pullspecs
+----
+
+[id="ocp-4-17-13-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when you clicked the `Don't show again` link in the Red Hat Ansible Lightspeed modal, it did not display the correct `General User Preference` tab when one of the other `User Preference` tabs was open. With this release, clicking the `Don't show again` link goes to the correct `General User Preference` tab. (link:https://issues.redhat.com/browse/OCPBUGS-48227[*OCPBUGS-48227*])
+
+* Previously, the installation program was not compliant with PCI-DSS/BAFIN regulations. With this release, the cross-tenant replication in {azure-first} is disabled, which reduces the chance of unauthorized data access and ensures strict adherence to data governance policies. (link:https://issues.redhat.com/browse/OCPBUGS-48119[*OCPBUGS-48119*])
+
+* Previously, if `SMTP smarthost` or `SMTP` from fields under `EmailConfigs` were not specified at the global or receiver level in Alertmanager configuration, it caused Alertmanager to crash because they are required fields. With this release, `prometheus-operator` fails reconciliation if these fields are not specified at the global or the receiver level, which avoids pushing an invalid configuration to Alertmanager. (link:https://issues.redhat.com/browse/OCPBUGS-48069[*OCPBUGS-48069*])
```

Comment:
Maybe something like this, as I felt like I had to read quite a ways before I understood the scope:
```suggestion
* Previously in Alertmanager configuration, if `SMTP smarthost` or `SMTP` from fields under `EmailConfigs` were not specified at the global or receiver level, it caused Alertmanager to crash because they are required fields. With this release, `prometheus-operator` fails reconciliation if these fields are not specified at the global or the receiver level, which avoids pushing an invalid configuration to Alertmanager. (link:https://issues.redhat.com/browse/OCPBUGS-48069[*OCPBUGS-48069*])
```